### PR TITLE
piper: 0.2.902 -> 0.2.903

### PR DESCRIPTION
**Pull request by tadeokondrak**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/os-specific/linux/piper/default.nix [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "piper-${version}";
```
Comment (dywedir) [inline pkgs/os-specific/linux/piper/default.nix]:
```suggestion
  - pname = "piper-${version}";
  + pname = "piper";
```

**Comment by Infinisil** [comment]
@GrahamcOfBorg eval

**Comment by xeji** [comment]
@GrahamcOfBorg build piper

**Comment by jtojnar** [comment]
https://github.com/libratbag/piper/compare/0.2.902...0.2.903

**Comment by ryantm** [comment]
@tadeokondrak thanks; please consider adding yourself as a maintainer.

**Pull request merged**